rails-snow-queen/app/assets/javascripts/quote.js

```diff
@@ -0,0 +1,63 @@
+class Quote {
+  constructor(config) {
+    this.errors = null;
+    this.onOpen = config.onOpen;
+    
+    this.handleErrors = this.handleErrors.bind(this);
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.addListeners();
+  }
+  
+  addListeners() {
+    $("#submitQuoteModal").on("show.bs.modal", this.handleOpenModal);
+    document.getElementById("new_quote").addEventListener("ajax:error", this.handleErrors);
+    document.getElementById("new_quote").addEventListener("ajax:success", this.handleSuccess);
+  }
+
+  handleOpenModal() {
+    let quoteData = this.onOpen();
+
+    let splitAddress = quoteData.geocodedAddress.split(",");
+    let addressNodeInModal = document.getElementById("addressModal");
+    addressNodeInModal.innerText = splitAddress;
+
+    let areaNodeInModal = document.getElementById("areaModal");
+    areaNodeInModal.innerText = `${quoteData.totalAreaInSqFt.toFixed(0)}`;
+    
+    let totalDueNodeInModal = document.getElementById("totalModal");
+    totalDueNodeInModal.innerText = `${quoteData.totalDue.toFixed(2)}`;
```

Comment:
These should have the `toLocaleString` stuff too

Reply:
I feel like a crazy person, I could swear I did this already